lighthouse/helpers/plates.py

```diff
@@ -117,6 +131,81 @@ def get_positive_samples(plate_barcode: str) -> Optional[List[Dict[str, Any]]]:
     return samples_for_barcode
 
 
+def row_is_normal_sample(row):
+    control_value = getattr(row, FIELD_DART_CONTROL)
+    return control_value is None or control_value == "NULL" or control_value == ""
```

Comment:
Do we need to check for all three of these? Shouldn't normal samples only take one of these values?